charts/newrelic-logging/templates/configmap.yaml

```diff
@@ -27,24 +27,42 @@ data:
         Skip_Long_Lines   On
         Refresh_Interval  10
 
-    [FILTER]
-        Name           record_modifier
-        Match          *
-        Record         cluster_name ${CLUSTER_NAME}
-
     [FILTER]
         Name           kubernetes
         Match          kube.*
         # We need the full DNS suffix as Windows only supports resolving names with this suffix
         # See: https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#dns-limitations
         Kube_URL       https://kubernetes.default.svc.cluster.local:443
         K8S-Logging.Exclude ${K8S_LOGGING_EXCLUDE}
+{{- if (include "newrelic.lowDataMode" .) }}
+        Labels         Off
+        Annotations    Off
+
+    [FILTER]
```

Comment:
Small data optimization: actually, given that we're doing this in our pipeline anyway, why don't we apply the lifting by default? The customer won't notice anything in the resulting data format, but we will end up transmitting less data over the wire.

Reply:
For me that's fine, I didn't want to change the default behaviour to limit the blast radius.

Reply:
As you prefer, no strong opinion here.